src/utils/parsers/settings.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @summary Replaces placeholder instances within a string with specified values.
+ * The placeholders are denoted as {n}, where n is the index of the value to be replaced in the array or arguments passed.
+ * If a placeholder has no corresponding value, the placeholder remains in the string.
+ * @param {String} str - the string to be formatted
+ * @param {String | String[]} values - to be put in the string
+ * @returns {String}
+ */
+export function formatString(str, ...values) {
+    return str.replace(/{(\d+)}/g, (match, index) => {
+        return typeof values[index] !== "undefined" ? values[index] : match;
+    });
+}
+
+const fortranDoubleRegex = "([-+]?\\d*\\.?\\d*(?:[Eed][+-]?\\d+)?)";
```

Comment:
Here's a good example of how RegEx should be defined and explained https://github.com/Exabyte-io/api-examples/blob/6ca21439ad2239bf82ade75ac0a34ccfd17bfcb0/examples/config.py#L34.
![Screen Shot 2023-06-20 at 1 15 14 PM](https://github.com/Exabyte-io/made.js/assets/721112/0d60c089-884c-474c-9959-cbb3549b5f9a)

Reply:
Makes sense. Noted.